app/src/lib/release-notes.ts

```diff
@@ -56,7 +56,14 @@ function notEmpty<TValue>(value: TValue | null | undefined): value is TValue {
 export function parseReleaseEntries(
   notes: ReadonlyArray<string>
 ): ReadonlyArray<ReleaseNote> {
-  return notes.map(n => parseEntry(n)).filter(notEmpty)
+  const pretext =
```

Comment:
The temporarily injected pretext note being markdown is failing unit test - but that is not a permanent change.